Get_a_name/99_bottles/99_bottles.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+for ((i=100
+	..; i>=1; i--));do
```

Comment:
@daphne4375 I'm getting a syntax error at this line. Did you mean to include the ..?